src/addons/transitions/ReactTransitionChildMapping.js

```diff
@@ -25,6 +25,9 @@ var ReactTransitionChildMapping = {
     if (!children) {
       return children;
     }
+    // TODO: `flattenChildren` now takes an extra `selfDebugID` argument
+    // for looking up the component stack in dev build.
+    // Need to figure out a way to get the `_debugID` here.
     return flattenChildren(children);
```

Comment:
@spicyj what's a good way to get and feed `debugID` to `flattenChildren` here? The only place that calls `ReactTransitionChildMapping.getChildMapping` (syntactic sugar for `flattenChildren`) is here: https://github.com/facebook/react/blob/master/src/addons/transitions/ReactTransitionGroup.js#L41. In other words, how do I/can I get a debugID from a component?